dataproc_jupyter_plugin/handlers.py

```diff
@@ -213,6 +213,8 @@ def full_path(name):
         "bigQueryPreview": bigquery.PreviewController,
         "bigQueryProjectsList": bigquery.ProjectsController,
         "bigQuerySearch": bigquery.SearchController,
+        "api/vertex/createJobScheduler": vertex.CreateController,
+        "api/vertex/createNewBucket": vertex.CreateNewBucketController,
```

Comment:
CreateNewBucketController to CreateBucketController.
change the api url to api/storage/createNewBucket